configapi/api_inventory.go

```diff
@@ -274,20 +273,64 @@ func handlePostUpf(c *gin.Context) error {
 	return nil
 }
 
-func handleDeleteUpf(c *gin.Context) error {
-	upfHostname, exists := c.Params.Get("upf-hostname")
-	if !exists {
-		errorMessage := "delete UPF request is missing upf-hostname"
-		logger.ConfigLog.Errorln(errorMessage)
-		return fmt.Errorf("%s", errorMessage)
+func updateGnbInNetworkSlices(gnbName string) error {
+	filterByGnb := bson.M{
+		"site-info.gNodeBs": bson.M{
+			"$elemMatch": bson.M{"name": gnbName},
+		},
 	}
-	logger.ConfigLog.Infof("received delete UPF %v", upfHostname)
-	msg := configmodels.ConfigMessage{
-		MsgType:     configmodels.Inventory,
-		MsgMethod:   configmodels.Delete_op,
-		UpfHostname: upfHostname,
+	rawNetworkSlices, errGetMany := dbadapter.CommonDBClient.RestfulAPIGetMany(sliceDataColl, filterByGnb)
+	if errGetMany != nil {
+		logger.DbLog.Errorln("failed to fetch network slices:", errGetMany)
```

Comment:
```suggestion
                logger.DbLog.Errorf("failed to fetch network slices: %v", errGetMany)
```
Using errorf to format log messages.